src/QsCompiler/CompilationManager/ProjectManager.cs

```diff
@@ -1291,6 +1310,14 @@ editCollection switch
                             codeActionSuggestions = codeActionSuggestions.Append(
                                 ("Update deprecated syntax in file.", file.GetWorkspaceEdit(formattingEdits)));
                         }
+
+                        // send telemetry if telemetry is enabled
+                        var telemetryProps = new Dictionary<string, string?>();
+                        telemetryProps["quantumSdkVersion"] = c.BuildProperties.SdkVersion?.ToString();
```

Comment:
This might not be needed. If the Extension version is supposed to be the same as QDK version, then we already get the version for every event emitted (every event includes the Extension version).

Reply:
The sdk version is the sdk version of the project that is being edited, not the version of the extension.